template/core/commons/src/main/java/io/bloco/core/commons/endpoints/OpenLibraryEndpoint.kt

```diff
@@ -0,0 +1,12 @@
+package io.bloco.core.commons.endpoints
+
+object OpenLibraryEndpoint {
+    val baseUrl: String
+        get() = "openlibrary.org"
+
+    fun search(keyword: String, fields: String = "title,key", limit: Int = 10) =
+        "/search.json?q=$keyword&fields=$fields&limit=$limit"
+
+    fun work(id: String) =
```

Comment:
Maybe you can also add specific fields in this request :$

Reply:
in the works endpoint if we do want more details we have to specify https://openlibrary.org/dev/docs/api/books, this its the other way around :/ 
since we only consume the title I think we can leave it as it is.